pytest_jira.py

```diff
@@ -337,8 +337,9 @@ def pytest_addoption(parser):
     # FIXME - Change to a credentials.yaml ?
     config = six.moves.configparser.ConfigParser()
     config.read([
-        '/etc/jira.cfg',
-        os.path.expanduser('~/jira.cfg'),
+        str(os.path.join('/', 'etc', 'jira.cfg')),
+        str(os.path.join(parser.extra_info['rootdir'], 'jira.cfg')),
```

Comment:
based on what I saw in tests logs, I think that `parser.extra_info['rootdir']` needs to get closed into `str()` ... but maybe I am wrong.

Reply:
Seems like you are right! I thought I had to remove `os.path.join` to make it work.